Cargo.toml

```diff
@@ -38,6 +38,9 @@ version = "1.0"
 [dependencies.base32]
 version = "0.4.0"
 
+[dependencies.crossterm]
+version = "0.17.8"
```

Comment:
I think this version has been yanked :-|

Reply:
Strange, `lib.rs` still shows that version.  I’ll open a bug report.